tutorials/user-defined-functions/standalone-etl-udf-deployment.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: flink.apache.org/v1beta1
+kind: FlinkDeployment
+metadata:
+  name: standalone-etl-udf
+spec:
+  image: quay.io/streamshub/flink-sql-runner-with-currency-converter:latest
```

Comment:
```suggestion
  image: quay.io/streamshub/flink-sql-runner-with-flink-udf-currency-converter:latest
```

Reply:
Are you actually publishing this image to our quay repo? If not you will need to add some CI to do that like we do with the data generator script.